android-json-form-wizard/src/test/java/com/vijay/jsonwizard/activities/JsonFormActivityTest.java

```diff
@@ -339,7 +357,7 @@ public void testToggleViewVisibilityShouldMakeViewInVisible() {
     }
 
     private JsonFormActivity getActivityWithIntent(Intent intent) {
-        controller = Robolectric.buildActivity(JsonFormActivity.class, intent).create().start();
+        controller = Robolectric.buildActivity(JsonFormActivity.class, intent).create();
```

Comment:
Why remove `start()`?

Reply:
It launches fragments that cause errors. We only need the activity reference for the tests to run.

Reply:
got it